changelog.d/11934.feature

```diff
@@ -0,0 +1 @@
+Expose the `type` property of a room in the _List Room_ and _Room Details_ API. 
```

Comment:
Heads up that you have a space at the end of this line, which would be why the lint check is failing.

Reply:
Thanks :)